webapp/channels/src/components/channel_bookmarks/utils.ts

```diff
@@ -124,8 +132,15 @@ export const useChannelBookmarks = (channelId: string) => {
         }
     }, [bookmarks]);
 
+    const reorder = async (id: string, prevOrder: number, nextOrder: number) => {
+        setTempOrder(insertWithoutDuplicates(order, id, nextOrder));
+        await dispatch(reorderBookmark(channelId, id, nextOrder));
```

Comment:
Does this need any sort of error handling, or do we think the chance of this failing is enough that we aren't going to bother?

I know I mentioned this to someone recently, but I'd love to add a simple toast system for cases like this where we want something that's not super-critical to be optimistic and there's no place in the UI for a proper error indicator for it

Reply:
Yes, we should add error handling. At a minimum the temp value should be cleared, a toast would be a nice addition.